components/tab/tab-title.tsx

```diff
@@ -0,0 +1,48 @@
+//タブのタイトル変更
+
+import { getCircleById } from "@/actions/circle/fetch-circle"
+
+const handlingTab = (key: string) => {
```

Comment:
`utils/format.ts`に`handlingTab`があるのでタブタイトルの変換ということで命名を変えてほしい。
```suggestion
const handlingTabTitle = (key: string) => {
```